### PR TITLE
[epoch] decrease internal voting power to 60%

### PR DESCRIPTION
**Pull request by LeoHChen**
EPOCH 432 starts around `Wed Jan 27 22:00 PDT 2021`

Signed-off-by: Leo Chen <leo@harmony.one>

### CODE REVIEW

**Comment by mindstyle85** [comment]
approved

**Comment by yelllowsin** [comment]
Sounds good to me, doing a small decrease as a start.

**Comment by deblanco** [comment]
LGTM!

**Comment by sophoah** [comment]
@LeoHChen I am good with the change as well

File: internal/configs/sharding/mainnet.go [hunk 1]
```diff
@@ -185,4 +188,5 @@ var (
 	mainnetV2_0 = MustNewInstance(4, 250, 170, numeric.MustNewDecFromStr("0.68"), genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1_5, mainnetReshardingEpoch, MainnetSchedule.BlocksPerEpoch())
 	mainnetV2_1 = MustNewInstance(4, 250, 130, numeric.MustNewDecFromStr("0.68"), genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1_5, mainnetReshardingEpoch, MainnetSchedule.BlocksPerEpoch())
 	mainnetV2_2 = MustNewInstance(4, 250, 90, numeric.MustNewDecFromStr("0.68"), genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1_5, mainnetReshardingEpoch, MainnetSchedule.BlocksPerEpoch())
+	mainnetV3_0 = MustNewInstance(4, 250, 90, numeric.MustNewDecFromStr("0.60"), genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1_5, mainnetReshardingEpoch, MainnetSchedule.BlocksPerEpoch())
```
Comment (rlan35) [inline internal/configs/sharding/mainnet.go:191]: tho it's a no-op. still please add to mainnetReshardingEpoch[]

**Comment by SmartStake** [comment]
I am good with the proposed change

**Comment by Chainode** [comment]
Looks good on this side. We can proceed with the change.

**Comment by staking4all** [comment]
Fine on my side. Approved

**Comment by xChickens** [comment]
👍

**Review by rlan35 — APPROVED** [review]

**Pull request merged**